### PR TITLE
fix naming convention

### PR DESCRIPTION
**Pull request by hd-deman**
now it's work as expected, but you guys need to update what yours linters return

```coffee
...
return new Promise (resolve, reject)->
   message = {type: 'error', message: 'Something went wrong', position:[[1,0], [1,1]], trace: []}
   resolve([message])
```
and we need update docs

### CODE REVIEW

File: lib/linter-view.coffee [hunk 1]
```diff
@@ -47,27 +47,27 @@ class LinterView
     ribbon = document.createElement 'span'
     ribbon.classList.add 'badge'
     ribbon.classList.add 'badge-flexible'
-    ribbon.classList.add 'badge-' + message.Type.toLowerCase()
-    ribbon.textContent = message.Type
+    ribbon.classList.add 'badge-' + message.type.toLowerCase()
+    ribbon.textContent = message.type
 
     theMessage = document.createElement('span')
-    if message.HTML and message.HTML.length
```
Comment (iam4x) [inline lib/linter-view.coffee]: I'm fine with `HTML`

**Comment by steelbrain** [comment]
Please note that this change is API breaking.

**Comment by AsaAyers** [comment]
That's fine, linter-plus hasn't been published so the only people running it should be the people running it. This is the time to make a breaking change like this :+1:

**Comment by iam4x** [comment]
let's keep `HTML` uppercase and I'm good with it

**Comment by AsaAyers** [comment]
I agree with `HTML` because it's an acronym.

**Pull request merged**